log/common.go

```diff
@@ -0,0 +1,36 @@
+package log
+
+import (
+	"io"
+	"os"
+)
+
+// For debugging purposes we can change the output-writer
+var stdOut io.Writer
+var stdErr io.Writer
+
+func init() {
+	stdOut = os.Stdout
+	stdErr = os.Stderr
+}
+
+const (
+	lvlWarning = iota - 20
+	lvlError
+	lvlFatal
+	lvlPanic
+	lvlInfo
+	lvlPrint
+)
+
+// Format indicates how the prints will be shown
+var Format = FormatLvl
```

Comment:
Ok i've done some research about logging, here's a good article [0].
It seems the most known third party library for logging (on github) seems very horrible [1], I don't much like it because it looks too complicated. I also want to keep the fact that we don't need to *instantiate* anything using the `dbg` library, using `logrus`, it seems we have to have the same control we need.
Actually it seems the default log library in Golang can support what we need [2], but unfortunatly, their levels are not quite clear, in fact they don't have any levels.

To me, since we already quite a good codebase in the `dbg` library, it does not seem a good idea to throw it away. But it definitely some refactoring so it's usable in both case (*debug view* and *client view* case) in a unified manner (and switchable on/off etc etc). Personally, I prefer the levels using clear words also: `Info Notice Warn Error Debug Fatal Panic` as `logrus` is doing (taken from their github page):
```
log.Debug("Useful debugging information.")
log.Info("Something noteworthy happened!")
log.Warn("You should probably take a look at this.")
log.Error("Something failed but I'm not quitting.")
// Calls os.Exit(1) after logging
log.Fatal("Bye.")
// Calls panic() after logging
log.Panic("I'm bailing.")
```

Theses are *classics* that everyone can understand and use the same way it's been used in other languages/libraries.

[0] http://dave.cheney.net/2015/11/05/lets-talk-about-logging
[1] https://github.com/Sirupsen/logrus
[2] https://golang.org/pkg/log/